src/ts/components/pages/EditProfile/EditProfile.tsx

```diff
@@ -419,9 +431,10 @@ export class EditProfile extends React.PureComponent<{},EditProfileState>{
     private chooseImage = (e: React.ChangeEvent<HTMLInputElement>) => {
         if(e.target.files !== null){
             if(
-                !( e.target.value.endsWith(".PNG")
-                || e.target.value.endsWith(".jpeg")
-                || e.target.value.endsWith(".jpg"))
+                !( e.target.value.toLowerCase().endsWith(".png")
+                || e.target.value.toLowerCase().endsWith(".jpeg")
+                || e.target.value.toLowerCase().endsWith(".jpg")
+                )
                 )
                 {
                     alert("We support only .PNG, .jpg and .jpeg files!");
```

Comment:
Could be a better/more generic message without the use of "we", since what is that referring to.

And it should maybe be put into the Json file.